### DISCUSSION TITLE
windows10debloater

### DISCUSSION THREAD

**Discussion by Carl-Gauss-root** [Sophia-Community/SophiApp#147] [Q&A] [answered]
Hi netizens, amantes del open source y el código limpio, tengo una pregunta, es la siguiente:
En conclusión: ¿qué tan eficaz y funcional es utilizar el windows10Debloater, para desinflar el sistema operativo?
Gracias

**Comment by farag2** [comment, marked as answer]
Hi. I do not know Spanish, I skipped at school. :) So better to use the international language. 👍 It's a little bit strange to ask such question here, but anyway, I will try to reply in detail. Not only me don't advice to use windows10Debloater due to that project is abandoned, the maintainer doesn't reply to any question/opened issue (look at its' count—256!!!). The code quality is very low. It provides functions that only make harm to system. So strongly recommend not use it ever. The maintainer had to archive the repo if he decided to abandon it as everyone does in such circumstance. Instead of this he just doesn't reply and ignores any questions.
